src/main/java/org/logstash/beats/BeatsHandler.java

```diff
@@ -54,8 +56,14 @@ public void channelRead0(ChannelHandlerContext ctx, Batch batch) throws Exceptio
 
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
+        InetSocketAddress remoteAddress = (InetSocketAddress) ctx.channel().remoteAddress();
```

Comment:
Ahh, nevermind. Java's SOcketChannel.getRemoteAddress can throw exceptions, but Netty's Channel.remoteAddress() will not (it returns null instead)